lib/elixir_boilerplate/config_helpers.ex

```diff
@@ -0,0 +1,53 @@
+# credo:disable-for-this-file CredoNaming.Check.Warning.AvoidSpecificTermsInModuleNames
+defmodule ElixirBoilerplate.ConfigHelpers do
```

Comment:
Since they are all environment helpers, should we just rename it to

```suggestion
defmodule ElixirBoilerplate.Environment do
```

Or something like that?

Reply:
@remi At the beginning, I extracted the module and kept the `Environment` name, but it felt weird to import functions from a module with a name too similar to `System` and `Application` from the stdlib.

Like the comment state, it's a set of helper functions for Elixir [`Config`](https://hexdocs.pm/elixir/Config.html), not an `Environment` set of function 🤔

Reply:
Alright, how about

```suggestion
defmodule ElixirBoilerplate.Config do
```

then?

Reply:
Or since they are helpers used in a single file, why extract them into an external module at all?

Reply:
The cognitive load of defining a module inside the (`runtime.exs`) script always felt weird imho. It pushes `runtime.exs` in a way that differs greatly from the other `Config` scripts and breaks it apart from the language conventions.

Reply:
And how about my previous comment? 🙂

Reply:
It's perfect, already made the change! Just haven't had time to push it yet…